.storybook/addons/codeEditorAddon/editor.js

```diff
@@ -251,10 +256,11 @@ export class EditorElement extends LitElement {
           )}
 
           <button
-            @click="${_ => generateProject(this.title, this.files)}"
+            @click="${_ => this.generateStackblitzProject()}"
```

Comment:
after refactoring the `generateStackblitzProject` function to an arrow function the `@click` directive can directly bind to it 
```suggestion
            @click="${this.generateStackblitzProject}"
```